front-end/src/app/_services/authentication.service.ts

```diff
@@ -48,10 +69,28 @@ export class AuthenticationService {
     });
   }
 
+  serverAuthLogout(id: string, accountId: string): Observable<any> {
 
+    let reqParams = {
+      token: {
+        id: localStorage.getItem(id),
```

Comment:
getItem(id) doesn't exist. I think it would be `localstorage.getItem('token').id`